sources/academy/tutorials/node_js/multiple-runs-scrape.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Scrape website from multiple Actor runs
+description: Learn how to run multiple instances of an Actor to scrape a website faster. This tutorial will guide you through the process of setting up your scraper.
+sidebar_position: 15.10
+slug: /node-js/multiple-runs-scrape
+---
+
+# Scraping website from multiple Actor runs
+
+**Learn how to run multiple instances of an Actor to scrape a website faster. This tutorial will guide you through the process of setting up your scraper.**
+
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+import MasterActorMainTs from '!!raw-loader!../../../../examples/ts-parallel-runs-scraping-master/src/main.ts';
+import MasterActorInputSchemaJson from '!!raw-loader!../../../../examples/ts-parallel-runs-scraping-master/.actor/input_schema.json';
+import ScraperActorMainTs from '!!raw-loader!../../../../examples/ts-parallel-runs-scraping-scraper/src/main.ts';
+import ScraperActorInputSchemaJson from '!!raw-loader!../../../../examples/ts-parallel-runs-scraping-scraper/.actor/input_schema.json';
+
+Imagine a large website that you need to scrape. You have a scraper that works well, but scraping the whole website is slow.
+You can speed up the scraping process by running multiple instances of the scraper in parallel.
+This tutorial will guide you through setting up your scraper to run multiple instances in parallel.
+
+> In a rush? You can check [full code example](https://github.com/apify/apify-docs/tree/master/examples/ts-parallel-runs-scraping-master) right away.
```

Comment:
```suggestion
:::note In a rush? 

You can check [full code example](https://github.com/apify/apify-docs/tree/master/examples/ts-parallel-runs-scraping-master) right away.

:::
```

I would suggest refraining from using codeblock this way, admonitions are much more elegant solution for this kind of notes.